packages/auto-label/src/helper.ts

```diff
@@ -125,6 +191,7 @@ export interface Config {
   };
   language?: LanguageConfig;
   staleness?: StaleConfig;
+  requestsize?: PullRequestSizeConfig;
```

Comment:
nit: casing feels weird, but we already have weird casing for `pullrequest` and `labelprefix`.

Reply:
Yeah, keeping it consistent